README.md

```diff
@@ -141,6 +142,11 @@ Valid options:
 - `SMALL2BIG`
 
 
+### `EXCLUDE_FILE` variable
+
+This variable is optional.  It allows you to exclude a file (e.g. index.html) from the file listings.
```

Comment:
Would it make sense to have this as EXCLUDE_FILES and have it be either a single string or an array of strings (or regexes?).